ballerina/tests/test.bal

```diff
@@ -140,6 +165,28 @@ function testCreate() {
     }
 }
 
+@test:Config {
+    enable: true
+}
```

Comment:
```suggestion
@test:Config {}
```

I think, by default it's set to true.

Reply:
Yes but this convention is used here so that a particular set of tests can be disabled at ones as we haven't used group by clause here.